CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+## [Unreleased]
+
+## [2.0.0] - 2017-07-24
+
+### Added
+
+* New rules:
+  * `rule-empty-line-before`
+  * `selector-max-universal`
+  * `at-rule-semicolon-space-before` (disabled)
+  * `function-url-scheme-blacklist` (disabled)
+  * `selector-max-attribute` (disabled)
+  * `selector-max-class` (disabled)
+  * `selector-max-combinators` (disabled)
+  * `selector-max-id` (disabled)
+  * `selector-max-type` (disabled)
```

Comment:
I wrote a little script to check if we were missing any rules. Seems to have found 2 rules that were missed: 7d2e52e

```
require('isomorphic-fetch');
var JSDOM = require('jsdom').JSDOM;
var globSync = require('glob').sync;
var path = require('path');

var root = path.resolve(__dirname);
var currentRules = [].concat.apply([], globSync(root + '/rules/*.js')
  .map((file) => Object.keys(require(file))));

  var stylelintRulesURl = 'https://stylelint.io/user-guide/rules/';

fetch(stylelintRulesURl, {method: 'GET'})
  .then((res) => {
    return res.text();
  })
  .then((html) => {
    var document = new JSDOM(html).window.document;
    var rulesNodeList = document.querySelectorAll('div.phenomic-BodyContainer ul li a:first-child ');

    var rulesList = Array.from(rulesNodeList)
    .map((rule) => {
      return {name: rule.textContent};
    });

    var missingRules = rulesList.filter((rule) => !currentRules.includes(rule.name));
    console.log('missingRules: ', missingRules);
  });
```